libyul/backends/evm/EVMDialect.cpp

```diff
@@ -109,6 +109,24 @@ pair<YulString, BuiltinFunctionForEVM> createFunction(
 	return {name, f};
 }
 
+set<YulString> createReservedIdentifiers()
+{
+	set<YulString> reserved;
+	for (auto const& instr: evmasm::c_instructions)
+	{
+		string name = instr.first;
+		transform(name.begin(), name.end(), name.begin(), [](unsigned char _c) { return tolower(_c); });
+		reserved.emplace(name);
+	}
+	reserved.emplace("linkersymbol");
+	reserved.emplace("datasize");
+	reserved.emplace("dataoffset");
+	reserved.emplace("datacopy");
+	reserved.emplace("setimmutable");
+	reserved.emplace("loadimmutable");
```

Comment:
Instead of emplacing several times, can we use `+` for set from `CommonData.h`?

Another suggestion would be converting the reserved for loop into an `applyMap`.